src/components/img.stories.tsx

```diff
@@ -50,7 +50,7 @@ const Placeholder: FC = () =>
             rawCaptionHtml: none,
             role: Role.Standard,
         }}
-        sizes="80vw"
+        sizes="40vw"
         className={some(css`width: 40vw; height: 30vw; display: block;`)}
```

Comment:
I think we can go further and extract this value. If your units are consistent you can do something like this:

```suggestion
        sizes="${displayWidth}"
        className={some(css`width: ${displayWidth}; height: ${displayWidth * ratio}; display: block;`)}
```

(Do not commit this change as I don't know whether you need to divide or multiply by `ratio`)

Reply:
I've extracted some of the props out and made this change using those values.